Library/Homebrew/language/python.rb

```diff
@@ -139,11 +139,34 @@ def virtualenv_create(venv_root, python = "python", formula = self)
         venv
       end
 
+      def needs_python?(py)
```

Comment:
:+1: Would you mind making the argument name `python` as this is more in line with other methods in this file?